### ISSUE TITLE
small crash game

### ISSUE THREAD

**Issue by etrigo1** [ZhooL/FS22_EnhancedVehicle#108]
Hello, I come to report a small bug that I'm finding. when I change implement "G" it causes a small crash in the game. I've only tested this mod and it's the cause of this micro crash. This small crash interferes with using a steering wheel to play the game.

**Comment by ZhooL** [comment]
Please provide your log.txt file

**Comment by etrigo1** [comment]
Hello, I used a blank save and only with this mod used. I also send a link to a video where the crash that happens when changing implements is noticeable. The crash is a small millisecond crash, but using a steering wheel to play causes the vehicle to go to "N" constantly.
I also created a video of what happens on the steering wheel.
With mods, the crash/delay is much more pronounced

https://mega.nz/file/XZZzhQqT#j_SWxdSCcx8E-Maq9SpQCwNuHUL80xcuUZzKZm0lwWU
https://mega.nz/file/PQYy0SDK#JMcj56GwIwkjAjIXFVznKlEbdzrZNlOhGN5wqUstX8E
https://mega.nz/file/bMBFXLKT#69fkeVVnsm4CEX7oBgsqtADbq-J6vLAfypSoLdUo83E

[log.txt](https://github.com/ZhooL/FS22_EnhancedVehicle/files/9578222/log.txt)
